app/controllers/api/v1/doctors_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Api::V1::DoctorsController < ApplicationController
+  before_action :set_doctor, only: %i[show update destroy]
+  def index
+    @doctors = Doctor.all
+    render json: @doctors
+  end
+
+  def show
+    render json: @doctor
+  end
+
+  def create
+    @doctor = Doctor.new(doctor_params)
+    if @doctor.save
+      render json: @doctor, status: :created
+    else
+      render json: @doctor.errors, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @doctor.update(doctor_params)
+      render json: @doctor
+    else
+      render json: @doctor.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @doctor.destroy
+    head :no_content
+  end
```

Comment:
- [ ] What happens here if the doctor is not successfully destroyed?  Kindly, add some error handling like you did in the update method (line 21). 👍